src/autocomplete/typeahead.js

```diff
@@ -39,6 +39,8 @@ function Typeahead(o) {
 
   this.css = o.css = _.mixin({}, css, o.appendTo ? css.appendTo : {});
   this.cssClasses = o.cssClasses = _.mixin({}, css.defaultClasses, o.cssClasses || {});
+  this.cssClasses.prefix =
+    o.cssClasses.formattedPrefix = _.formatPrefix(this.cssClasses.prefix, this.cssClasses.noPrefix);
```

Comment:
I save the formattedPrefix in the options that are then passed down to other components. Is it fine?